### PR TITLE
changed gif image

### PR DESCRIPTION
**Pull request by maridoroshuk**
#### Summary

Changed gif image in accordance to text (Closes issue #13702) 
#### Motivation

> The image shows a [Higher-order curves](https://en.wikipedia.org/wiki/B%C3%A9zier_curve#Higher-order_curves) while the text describes a [Quadratic curves](https://en.wikipedia.org/wiki/B%C3%A9zier_curve#Quadratic_curves). This thing can create confusion in the readers.
#### Metadata

- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by rebloor** [comment]
@maridoroshuk I'm assuming I've been added here because of the changes to move the Examples/Browser compatibility sections. However, were these changes intended as they're not mentioned in the description? I'm probably not the best person to comment on the grocery change.

File: files/en-us/mozilla/add-ons/webextensions/api/browsingdata/remove/index.md [hunk 1]
```diff
@@ -87,6 +83,10 @@ browser.browsingData.remove({},
 then(onRemoved, onError);
 ```
 
+## Browser compatibility
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/browsingdata/remove/index.md:86]: This section needs to be moved below the {{WebExtExamples}} who is well here and on the other pages.

**Review by rebloor — CHANGES_REQUESTED** [review]

**Comment by maridoroshuk** [comment]
> @maridoroshuk I'm assuming I've been added here because of the changes to move the Examples/Browser compatibility sections. However, were these changes intended as they're not mentioned in the description? I'm probably not the best person to comment on the grocery change.

@rebloor  Yes, sorry! It has to be my oher PR (https://github.com/mdn/content/pull/13651)

**Comment by rebloor** [comment]
@maridoroshuk thanks, I see your other PR has been merged, could you do a new one to keep the examples together

**Pull request closed without merging**